wormhole-connect/vite.config.ts

```diff
@@ -13,6 +14,18 @@ const packagePath = __dirname.endsWith('wormhole-connect')
   : './wormhole-connect/package.json';
 const { version } = require(packagePath);
 
+let gitHash = 'unknown';
+try {
+  gitHash = execSync("git log | head -n 1 | awk '{ print $2 }'").toString();
```

Comment:
Is the second field that `awk` prints always the commit hash? Another way is `git log -1 --format=%H` but I defer

Reply:
that's cleaner because it doesn't depend on `head` and `awk`. updated.